### PR TITLE
Increase graphic container width to show bigger numbers

### PR DESCRIPTION
**Pull request by Iinh**
This PR increases the width of the DataGraphic container in order to show bigger values. See original PR in `graph-paper`: https://github.com/graph-paper-org/graph-paper/pull/365 for the before and after.

As discussed with @robhudson, I copied the DataGraphic component from graph-paper to make the width change directly in GLAM. This is the start of an effort to pull all graph-paper components back into glam.

### CODE REVIEW

File: src/components/explore/AggregationComparisonGraph.svelte [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <script>
   import { Axis } from '@graph-paper/guides';
 
-  import { DataGraphic } from '@graph-paper/datagraphic';
-
   import { tooltip as tooltipAction } from '@graph-paper/core/actions';
+  import DataGraphic from '../../graph-paper/datagraphic/DataGraphic.svelte';
```
Comment (robhudson) [inline src/components/explore/AggregationComparisonGraph.svelte]: I think we can drop the `graph-paper/` folder here. Or do you feel there's a reason for it?

File: package.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
     "svelte-loader": "3.1.2"
   },
   "dependencies": {
+    "d3-array": "^3.0.1",
```
Comment (robhudson) [inline package.json]: Let's drop the `^` to match the rest.

File: src/graph-paper/datagraphic/DataGraphic.svelte [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+    justify-content: center;
+  }
+</style>
+
+<div
+  bind:this={container}
+  class="data-graphic-container"
+  style="width: {$graphicWidth}px; height: {$graphicHeight}px;">
+  <svg
+    style="width: {$graphicWidth + 90}px; height: {$graphicHeight}px;"
```
Comment (robhudson) [inline src/graph-paper/datagraphic/DataGraphic.svelte]: Just noting that here is the change from the original DataGraphic.svelte.

File: src/components/explore/AggregationComparisonGraph.svelte (hunk 1 above)
Comment (Iinh) [reply src/components/explore/AggregationComparisonGraph.svelte]: I intend to use this `graph-paper/` folder for all the components that we're going to pull. Wdyt?

Comment (robhudson) [reply src/components/explore/AggregationComparisonGraph.svelte]: Okay. I think "graph-paper" doesn't really have meaning inside glam. And it's an empty folder. I think the graph-paper components should all live with the other components inside `src/components` and subfolders there.

Comment (Iinh) [reply src/components/explore/AggregationComparisonGraph.svelte]: Makes sense. I'll move DataGraphics to `src/components/` then

**Review by robhudson — APPROVED** [review]

**Pull request merged**